### PR TITLE
Move crio and kubernetes jobs from rosie

### PR DESCRIPTION
**Pull request by bbguimaraes**
@kargakis @stevekuznetsov

https://github.com/openshift/aos-cd-jobs/pull/1230

### CODE REVIEW

File: cluster/ci/config/prow/config.yaml [hunk 1]
```diff
@@ -1469,3 +1469,13 @@ periodics:
     - name: token
       secret:
         secretName: openshift-bot-token
+- name: ci-kubernetes-conformance-node-e2e-rhel
+  agent: jenkins
+  labels:
+      master: ci.openshift.redhat.com
+  interval: 12h
```
Comment (ingvagabund) [inline cluster/ci/config/prow/config.yaml]: What about each 2 hours? Or is it to often?

**Comment by ingvagabund** [comment]
This needs to be merged ASAP. We no longer run the upstream node e2e CI over containerized kubelet. That's too bad.

**Comment by ingvagabund** [comment]
@sjenning @derekwaynecarr  FYI

File: cluster/ci/config/prow/config.yaml (hunk 1 above)
Comment (stevekuznetsov) [reply cluster/ci/config/prow/config.yaml]: That was the original schedule, right? So let's use 2h

Comment (ingvagabund) [reply cluster/ci/config/prow/config.yaml]: Yes sir

Comment (bbguimaraes) [reply cluster/ci/config/prow/config.yaml]: My bad.  Long time no crontab.

Fixed.

**Comment by ingvagabund** [comment]
Thanks guys

**Pull request merged**